crates/pathfinder/src/p2p_network/sync_handlers.rs

```diff
@@ -212,15 +212,18 @@ fn classes(
             break;
         };
 
+        // This is a temporary measure to avoid exceeding the max size of a protobuf message.
+        let class = zstd::bulk::compress(&class, 19)?;
```

Comment:
In general, using level 19 is a bad idea. Can't we just use the default (that is, specify 0 as the level which means the default)?

Reply:
:cold_sweat: yeah 19 is..bad, will use the default ofc